tests/e2e/util.go

```diff
@@ -1870,6 +1871,64 @@ func getWCPSessionId(hostname string, username string, password string) string {
 
 }
 
+// getk8sWindowsWorkerIP returns the windows worker IP for given nodeName
+func getk8sWindowsWorkerIP(ctx context.Context, client clientset.Interface, nodeName string) string {
+	var err error
+	nodes, err := client.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	var windowsWorkerIp string
+	for _, node := range nodes.Items {
+		if node.Name == nodeName {
+			addrs := node.Status.Addresses
+			for _, addr := range addrs {
+				if addr.Type == v1.NodeInternalIP && (net.ParseIP(addr.Address)).To4() != nil {
+					windowsWorkerIp = addr.Address
+				}
+			}
+		}
+	}
+	gomega.Expect(windowsWorkerIp).NotTo(gomega.BeEmpty(), "Unable to find k8s windows worker IP")
+	return windowsWorkerIp
+
+}
+
+// getWindowsDiskSize finds the windowsWorkerIp and returns the size of the volume
+func getWindowsDiskSize(client clientset.Interface, pod *v1.Pod) (int64, error) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	nodeName := pod.Spec.NodeName
+	windowsWorkerIP := getk8sWindowsWorkerIP(ctx, client, nodeName)
+	framework.Logf("windows worker ip %s", windowsWorkerIP)
+	var err error
+	var size string
+	nimbusGeneratedVcPwd := GetAndExpectStringEnvVar(nimbusVcPwd)
+	sshClient, err := simplessh.ConnectWithPassword(windowsWorkerIP, windowsUser, nimbusGeneratedVcPwd)
```

Comment:
Directly add nimbusVcPwd to the ssh connection

Reply:
we have to read env variable..right?? How can we read nimbusVcPwd directly?